src/components/Chip.tsx

```diff
@@ -27,3 +33,17 @@ export function Chip<X extends Only<Xss<Margin>, X>>(props: ChipProps<X>) {
     </span>
   );
 }
+
+function getStyles(type?: TagType) {
```

Comment:
Thought: This is just fine, I write `switch`es like this a lot; fwiw another way we also do sometimes is:

```
const typeStyles: Record<TagType, Properties> = {
  [TagType.Caution]: Css.byYellow200.$,
  ...
}

// and in
...typeStyles[type ?? TagType.Neutral];
```

It might be a little bit shorter here.

Granted, both `switch` + `Record` idioms provide exhaustive "if you add a new type, you need to add a new map/case" checking from the compiler, so it's up to you, just mentioning it.

Reply:
Just updated, I prefer to use a `Record` here but was following Tag which used a switch and didn't want to throw things off so thanks for the info.